### ISSUE TITLE
How do I build this?

### ISSUE THREAD

**Issue by NovenTheHero** [tonyp7/MegaDumper#1]
I am a total n00b to this, but I love this and would love to get one built. How would I actually go about sending this off to get made?

**Comment by birdybro** [comment]
Yeah, the readme suggests there is a BOM in /hardware/ but there isn't. Maybe it was removed?

**Comment by NovenTheHero** [comment]
> Yeah, the readme suggests there is a BOM in /hardware/ but there isn't. Maybe it was removed?

Yeah, thanks for affirming my confusion =).
